BackEnd/src/main/kotlin/quest/marketstack/tradingapp/model/ClosedTrade.kt

```diff
@@ -0,0 +1,82 @@
+package quest.marketstack.tradingapp.model
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document(collection = "\${api.closed-trade.collection}")
```

Comment:
Same as the baseUrl changes, moved this to application.yml. Better practice to have it there